react/features/chat/components/ChatMessage.web.js

```diff
@@ -61,7 +62,22 @@ class ChatMessage extends PureComponent<Props> {
         const escMessage = messagetoDisplay.replace(/</g, '&lt;')
             .replace(/>/g, '&gt;')
             .replace(/\n/g, '<br/>');
-        const messageWithHTML = processReplacements(escMessage);
+        const processedMessage = [];
+        const items = toArray(escMessage, { className: 'smiley' });
+
+        items.forEach(i => {
```

Comment:
You can use a map to express iterating over items and returning something for each. 

Do you think it's obvious enough that toArray returns the string split at potential smileys? I'm a little concerned that "items" is a vague name for the return of toArray. Maybe this is more a criticism of toArray, though.

Reply:
This is basically an example code about Linkify I was given and I used. I will try now to replace it with something more readable.